src/shared/components/help-popup/HelpPopupBody.tsx

```diff
@@ -42,6 +42,18 @@ const HelpPopupBody = (props: Props) => {
   const { article, loadingState } = useHelpArticle(currentReference);
   const historyRef = useRef<HelpPopupHistory | null>(null);
 
+  useEffect(() => {
+    const trackHelpArticleViewed = new CustomEvent('analytics', {
+      detail: {
+        category: 'contextual-help',
+        action: article?.type + '_article_opened',
```

Comment:
- As written, this code will dispatch some events even when there is no article. What would this mean?
- Why are we tracking this independently from "contextual-help/opened" in `HelpPopupButton`? Why is it not a single event? If contextual help opens, what else can it do other than display an article?

Reply:
>As written, this code will dispatch some events even when there is no article. What would this mean?

I've added an `if` wrapper to check if an article exists